### PR TITLE
Refactored facebook APP ID into an env variable

### LINKED ISSUE #1321: Consider making FB_APP_ID an environment variable
Currently stored as an inline constant in our Helmet component for NoRent: https://github.com/JustFixNYC/tenants2/blob/a85dbfbbbb2d1334e651b7b01a17bb27a23d501c/frontend/lib/norent/components/helmet.tsx

### PR DESCRIPTION
**Pull request by sraby**
This PR fixes #1321 by making our Facebook APP Id an optional environmental variable instead of having it hard coded in to our Helmet component on the Org Site. Now, it will also be easier for us to reuse in the regular tenant app if we need to.

### CODE REVIEW

**Comment by sraby** [comment]
@toolness we're gonna want to make sure to configure this env variable in Heroku once this is merged, right?

File: frontend/lib/app-context.tsx [hunk 1]
```diff
@@ -168,6 +168,13 @@ export interface AppServerInfo {
    * intended for training or review purposes.
    */
   isDemoDeployment: boolean;
+
+  /**
+   * An optional "Facebook App ID" for the an associated organization's Facebook page.
+   * If empty (the default), there will be no reference to a Facebook App ID
+   * in the header metatags for the site.
+   */
+  facebookAppId?: string;
```
Comment (toolness) [inline frontend/lib/app-context.tsx:177]: Based on the code it looks like this will never actually be `undefined`, so maybe remove the `?`?  That way we'll be sure that it is either an empty string or a populated string, rather than possibly undefined, or an empty string, or a populated one (and less cases == less edge cases == less future headaches lol).  If you do that though, TypeScript will likely complain that the `FakeServerInfo` object in `frontend/lib/tests/util.tsx` doesn't have an entry for it, so you'll want to set it to the empty string there!

**Comment by toolness** [comment]
> @toolness we're gonna want to make sure to configure this env variable in Heroku once this is merged, right?

Yup that is true!  Do you want to do that for production or should I?  And we could also set it on demo but I'm not sure if that will confuse our analytics or not.

File: frontend/lib/app-context.tsx (hunk 1 above)
Comment (sraby) [reply frontend/lib/app-context.tsx]: Noice! All set
